vm/os-windows.cpp

```diff
@@ -296,6 +296,50 @@ void handle_ctrl_c() {
   SetConsoleCtrlHandler(factor::ctrl_handler, TRUE);
 }
 
+const int ctrl_break_sleep = 10; /* msec */
```

Comment:
@mrjbq7: "Also, it is unclear how it works when running two Listeners."

Two listeners can't be supported, unfortunately, because it's the VM that gets interrupted, not a specific Factor coop-thread.